_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,267 @@
+
+```html
+<body onload="startUpdates()">
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" onclick="stopUpdates()" value="Stop updates" />
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+#### Passed Example 2
+
+The text content of the `span` element automatically updates after the page completes loading. A button is available to pause and resume the automatic updates. The rule is not applicable to the first `p` element because it has a child (the `span` element) whose content updates.
+
+```html
+<body onload="startUpdates()">
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" id="control" onclick="toggleUpdates()" value="Pause updates" />
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+#### Passed Example 3
+
+The text content of the `span` element automatically updates after the page completes loading. A button is available to hide the automatically updating content. The rule is not applicable to the first `p` element because it has a child (the `span` element) whose content updates.
+
+```html
+<body onload="startUpdates()">
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" onclick="hide()" value="Hide updating content" />
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+#### Passed Example 4
+
+The text content of the `span` element automatically updates after the page completes loading. A [mechanism][] is available to change the update frequency. The rule is not applicable to the first `p` element because it has a child (the `span` element) whose content updates.
+
+```html
+<body onload="startUpdates()">
+	<p>Random number: <span id="target">1</span></p>
+	<label for="interval">Content update frequency (seconds):</label>
+	<input type="text" id="interval" />
+	<input type="button" onclick="changeFrequency(document.getElementById('interval').value)" value="Change frequency" />
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+### Failed
+
+#### Failed Example 1
+
+The text content of the `span` element automatically updates after the page completes loading. There is no [mechanism][] to stop, pause, hide or alter the frequency of the automatic updates.
+
+```html
+<body onload="startUpdates()">
+	<p>Random number: <span id="target">1</span></p>
+
+	<script type="text/javascript" src="/test-assets/efbfc7/script.js"></script>
+</body>
+```
+
+#### Failed Example 2
+
+The text content of the `span` element automatically updates after the page completes loading. A button is available to stop the automatic updates, but the button is not visible.
+
+```html
+<body onload="startUpdates()">
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" onclick="stopUpdates()" value="Stop updates" style="display:none" />
```

Comment:
I would advice picking a different method of making this text invisible because using display:none also places the text outside of the accessibility tree.